### DISCUSSION TITLE
Welcome to Bet365Bot Discussions!

### DISCUSSION THREAD

**Discussion by JDaniloC** [JDaniloC/Individual-Bet365Bot#8] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Let's do bot work to new version of Bet365 and in other's browsers
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by guedezvaldo** [comment]
Fala Danilo, tudo bem? Tem alguma atualização do bot? estou tendo problema para utilizar o bot segui todo o passo a passo e estou tendo o seguinte erro
![image](https://user-images.githubusercontent.com/104527901/179866179-4b7232d1-13cd-47cc-8238-2b3228f3851f.png)
![image](https://user-images.githubusercontent.com/104527901/179866205-58f49106-988e-4c6d-b348-29cc63d78189.png)

Sou novo em python onde estou errando

**Reply by JDaniloC** [reply]
Ele tá dando erro na conexão com o MongoDB (O banco de dados), você pode remover essa conexão editando o código. De qualquer jeito, presumo que você não seguiu o passo de adicionar um .env com a sua conexão com o MongoDB.

**Reply by guedezvaldo** [reply]
![image](https://user-images.githubusercontent.com/104527901/180092861-d9c05751-a5e4-4025-adbb-9f7bb057e3cb.png)
Eu havia adicionado sim :( mas obrigado. sem os<>

**Reply by JDaniloC** [reply]
Man, você não preencheu as informações corretamente. Não tem esse <>, era só um indicativo de onde colocar.

**Comment by pedrim01** [comment]
Boa noite Danilo!

Tentando realizar um bot para realizar aposta automática na bet365, usando js, selenium em py, playwright botcity, entre outros frameworks, percebi que a bet365 bloqueia ao clicar na probabilidade da aposta, informando que a página foi suspensa, após a abertura do pop up.

Esse problema, já é possível de ser contornado?

**Reply by JDaniloC** [reply]
Boa noite! Desculpa a demora, mas faz muito tempo que não faço mais nada em relação a essa plataforma, não sei te ajudar. Boa sorte.

**Reply by pedrim01** [reply]
Grato! 

Conhece alguém que poderia ajudar?

**Reply by JDaniloC** [reply]
Não... Mas o mercado é bem aquecido, muitos tentam me pagar para fazer algo na Bet, e bem... Se tem quem contrate, então deve haver outros desenvolvedores.

**Reply by BigearlYOW** [reply]
Boa tarde Danilo. Estaria interessado em dar continuidade nesse projeto? 

Meu intuinto e ate um pouco diferente. Trata-se de receber "entradas" que vem do Telegram e aplica-las. Retirando o fator emocional e erro humano que podem causar perdas.

**Reply by JDaniloC** [reply]
Não, mas agradeço a confiança. O projeto está disponível para qualquer programador utilizar como referência.

**Comment by Z-Fir022T** [comment]
Olá, estou interessado em um Bot para bet365, seria algo não tão complexo mas seria para futebol no mercado 1X2

**Reply by JDaniloC** [reply]
Eu estou disponível para esse tipo de coisa, agradeço a confiança :)
